packages/workflow/src/LoggerProxy.ts

```diff
@@ -0,0 +1,45 @@
+import { 
+	ILogger, 
+	LogTypes,
+} from './Interfaces';
+
+
+let instance: ILogger | undefined;
+
+export function init(logger: ILogger) {
+	instance = logger;
+}
+
+export function getInstance(): ILogger {
```

Comment:
To make this more explicit than just "instance":

```suggestion
export function getLogger(): ILogger {
```